fabulaws/ubuntu/instances/base.py

```diff
@@ -76,6 +77,32 @@ def _encrypt_device(self, device, passwd=None):
         return device
 
     @uses_fabric
+    def _mount_and_persist(self, device, mount_point, fs_type=None, opts=None):
+        mount_cmd = ' '.join([
+            'mount',
+            '-o {}'.format(opts) if opts else '',
+            '-t {}'.format(fs_type) if fs_type else '',
+            device,
+            mount_point,
+        ])
+        fstab_entry = ' '.join([
+            device,
+            mount_point,
+            fs_type if fs_type else self.fs_type,
+            opts if opts else 'defaults',
+            '0',
+            '0',
+        ])
+        sudo(mount_cmd)
+        if self.fs_encrypt:
+            if not files.exists(self.mount_script):
+                files.append(self.mount_script, "#!/bin/sh", use_sudo=True)
```

Comment:
From my reading of the docs for [append](http://docs.fabfile.org/en/1.12/api/contrib/files.html#fabric.contrib.files.append), we don't really need to guard for whether or not the file exists (nor for the `chmod` statement). Not a big deal, but I'd be in favor of one less `if` statement.